app/components/follow_button.rb

```diff
@@ -1,23 +1,40 @@
 # frozen_string_literal: true
 
-class FollowButtonComponent < ViewComponent::Base
+class Components::FollowButton < Components::Base
+  include Phlex::Rails::Helpers::ButtonTo
+
   def initialize(follower:, target:, name: nil)
     @signed_out = follower.nil?
     @target = target
     @following = follower&.following? target
     @name = name
   end
 
-  def before_render
+  def view_template
+    return unless render?
```

Comment:
Nice @Floppy. It's great to see more people using Phlex. 

BTW, Phlex, by default, doesn't render the component if the `#render?` method returns false, so this is not necessary.

```ruby
def view_template
  h1 { "Not rendered" }
end

def render? = false
```

I think this info is missing from Phlex docs.

Reply:
Ah, fantastic, I had it with ViewComponent, but didn't see the same API for Phlex - good to know it's there! I'll remove my extra checks :)